### PR TITLE
bootstrap gygdev service and drone integration

### PR DESCRIPTION
**Pull request by pvonr**
Hi guys,

here the bootstrap version of a hello-world flask app in gygdev. 

setting up gygdev (after it is installed)

1. change ~/.gygdev.yml to (replace the config stuff to match your env but keep the skip_vagrant flag)
```
config:
  chef_path: ~/repos/chef
  id_rsa: ~/.ssh/id_rsa
  ssh_user_for_production: pascal
  workhome: ~/repos/gyg
  skip_vagrant: true
services:
  sem-ad:
    repo: git@github.com:getyourguide/sem-ad.git
    develop: true
    branch: gygdev
```

2. run `gygdev init` inside the `sem-ad` repo
3. run `gygdev up`
4. run `gygdev info` to see that we have a `web` container up and running for the `sem-ad` service
5. run `gygdev tunnel sem-ad web 5000:5000` to expose the flask application to your host system
6. go to http://localhost:5000 in your browser and say hello to your first gygdev microservice ;)

I did not include stuff that @olel-gyg  did yet. This will come next.

Cheers,
Pascal

### CODE REVIEW

**Comment by pvonr** [comment]
FYI @damienpalacio @alexandruionita-gyg this is how a simple flask application could look like within our new gygdev environment

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea
+.gygdev_generated
+docker-compose.yml
+.drone.yml
```
Comment (andrebco) [inline .gitignore]: Just a small comment. I normally leave the last line to normalise the doc. You could set your editor to automatically do it.

Comment (pvonr) [reply .gitignore]: thanks for that. Feel free to do it :)

**Comment by damienpalacio** [comment]
Nice! Thanks Pascal!!

**Pull request merged**